Assets/Scripts/Models/Area/World.cs

```diff
@@ -478,6 +484,7 @@ private void TickEveryFrame(float deltaTime)
     {
         CharacterManager.Update(deltaTime);
         FurnitureManager.TickEveryFrame(deltaTime);
+        GameEventManager.Update(deltaTime);
         ShipManager.Update(deltaTime);
```

Comment:
It was updated every frame before. So I went with that. We can always update less, but not sure how good it will work.

@dusho You can do that in another PR.